components/core/src/clp/regex_utils/ErrorCode.cpp

```diff
@@ -0,0 +1,78 @@
+#include "regex_utils/ErrorCode.hpp"
+
+#include <string>
+#include <string_view>
+#include <system_error>
+
+using std::error_category;
+using std::error_code;
+using std::string;
+using std::string_view;
+
+namespace clp::regex_utils {
+
+/**
+ * Class for giving the error codes more detailed string descriptions.
+ * This class does not need to be seen outside the std error code wrapper implementation.
```

Comment:
This sentence (line 16) is a pointless without elaboration. Either explain why or remove it. If it is standard implementation/practice for error_category then removing it is probably fine.